### PR TITLE
Blog post draft

### PR DESCRIPTION
**Pull request by tony-sull**
[Draft post](https://deploy-preview-448--astro-www-2.netlify.app/blog/themes-catalog-updates/)
[Themes catalog updates preview](https://next.astro.build/themes/)

### CODE REVIEW

File: src/content/blog/themes-catalog-updates.mdx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+## Premium Themes
+
+Building and maintaining a theme is a massive undertaking - from design and development to testing and user support the work really piles up fast.
+
+<BlogContentImage
+    src={paidThemes}
+    alt="Screenshots showing the details page for a paid theme on the new themes catalog"
+/>
+
+The catalog now supports premium themes, allowing theme authors to include a link to where the theme can be purchased. Whether a theme author uses [Gumroad](https://gumroad.com/) or sells on their own site, our catalog is making it easier to discover new themes **without** acting as a marketplace or taking a cut of sales.
```
Comment (delucis) [inline src/content/blog/themes-catalog-updates.mdx:53]: What do you think about not calling out a shady thing that we’re not doing? Not sure we need to float something negative when we want people to focus on the positive.
```suggestion
The catalog now supports premium themes, allowing theme authors to include a link to where the theme can be purchased. Whether a theme author uses [Gumroad](https://gumroad.com/) or sells on their own site, our catalog makes it easier to share your theme and have Astro users discover it.
```

File: src/content/blog/themes-catalog-updates.mdx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+## New Submission Form
+
+Submitting your theme to the catalog just got easier.
+
+<BlogContentImage
+    src={submitThemes}
+    alt="An online form used to submit theme details with preview images, descriptions, and tags."
+/>
+
+Gone are the days of filling out GitHub issue templates - the new submission form that makes it simple to upload multiple preview images, select the tools used in your theme, and even includes a rich text editor for writing the full description of your theme.
```
Comment (delucis) [inline src/content/blog/themes-catalog-updates.mdx:64]: Nit: stray “that” and I think these could be two sentences for that extra punchiness ✨ 

```suggestion
Gone are the days of filling out GitHub issue templates. The new submission form makes it simple to upload multiple preview images, select the tools used in your theme, and even includes a rich text editor for writing the full description of your theme.
```

File: src/content/blog/themes-catalog-updates.mdx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+/>
+
+## Theme Detail Pages
+
+The initial release of our themes catalog focused on creating a launchpad for Astro themes, giving a quick overview of themes with a preview image and short description. As the list of themes grew that lead to an obvious question, what if you want to know more about a specific theme?
+
+<BlogContentImage
+    src={themesPreview}
+    alt="Screenshots of the new Astro themes catalog details page"
+/>
```
Comment (delucis) [inline src/content/blog/themes-catalog-updates.mdx:36]: Looks like this is the same image as immediately above. Should they be different?

image

File: src/content/blog/themes-catalog-updates.mdx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+Theme authors can now share more details about the theme they worked so hard to design, develop, and test. Multiple images can be included to show off different page templates, dark mode support, and included components. Badges add a quick reference to what tools and frameworks were used to build theme. Most importantly, a full description section means theme authors can share details that would never fit in a card layout.
+
+<Note title="Submit your theme">
+  Already have a theme in our catalog? Check out our new [submission form](https://astro.build/themes/submit/) to update your theme's details!
+</Note>
+
+## Premium Themes
+
+Building and maintaining a theme is a massive undertaking - from design and development to testing and user support the work really piles up fast.
```
Comment (delucis) [inline src/content/blog/themes-catalog-updates.mdx:46]: Punctuation nit:

```suggestion
Building and maintaining a theme is a massive undertaking — from design and development to testing and user support, the work really piles up fast.
```

File: src/content/blog/themes-catalog-updates.mdx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+<BlogContentImage
+    src={paidThemes}
+    alt="Screenshots showing the details page for a paid theme on the new themes catalog"
+/>
+
+The catalog now supports premium themes, allowing theme authors to include a link to where the theme can be purchased. Whether a theme author uses [Gumroad](https://gumroad.com/) or sells on their own site, our catalog is making it easier to discover new themes **without** acting as a marketplace or taking a cut of sales.
+
+## New Submission Form
+
+Submitting your theme to the catalog just got easier.
```
Comment (delucis) [inline src/content/blog/themes-catalog-updates.mdx:57]: I’d be tempted to throw in an exclamation mark here but I guess that’s a stylistic choice!

```suggestion
Submitting your theme to the catalog just got easier!
```

**Review by delucis — COMMENTED** [review]
This looks great @tony-sull! One visual nit is that the text “Themes” is not vertically centered in the hero image but obviously not crucial!

image

File: src/content/blog/themes-catalog-updates.mdx (hunk 3 above)
Comment (tony-sull) [reply src/content/blog/themes-catalog-updates.mdx]: Yep, that should have been a separate image.  Thanks!

**Pull request merged**